gym/wrappers/record_episode_statistics.py

```diff
@@ -7,11 +7,98 @@
 import gym
 
 
+class ClassicStatsInfo:
```

Comment:
Would it be possible to inherit from `VecVenStatsInfo` then only override the `add_episode_statistics`? This seem like the only difference

Update name to `VecEnvClassicStatsInfo`

Reply:
I made `VecEnvStatsInfo` inherit from `StatsInfo` which can be seen more as the "base" class for non vectorized environments.
Renamed to `StatsInfo`  instead of `VecEnvClassicStatsInfo` because it is used when the wrapped env is not vectorized. I think `Classic` word can be removed here because we do not have this differentiation in non vectorized envs.